src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetAsciiTests.cs

```diff
@@ -83,24 +83,22 @@ public void GetAscii(string unicode, int index, int count, string expected)
             }
             Assert.Equal(expected, new IdnMapping().GetAscii(unicode, index, count));
         }
-        
+
         [Fact]
-        public void TestGetAsciiWithDot()
+        [PlatformSpecific(TestPlatforms.AnyUnix ^ TestPlatforms.OSX)]
```

Comment:
Nit: we've used the logic:
```
TestPlatforms.AnyUnix & ~TestPlatforms.OSX
```
elsewhere.  I realize it's the same thing, but I think this reads a bit more easily, reading at least to me as "any unix and not OSX", whereas I have to stop and think for a moment with the xor.